test/server/live-test.js

```diff
@@ -0,0 +1,364 @@
+const expect               = require('chai').expect;
+const noop                 = require('lodash').noop;
+const Promise              = require('pinkie');
+const path                 = require('path');
+const createTestCafe       = require('../../lib/index');
+const Controller           = require('../../lib/live/controller');
+const FileWatcher          = require('../../lib/live/file-watcher');
+const LiveModeRunner       = require('../../lib/live/test-runner');
+const LiveModeBootstrapper = require('../../lib/live/bootstrapper');
+
+
+const fileName1 = path.resolve('test/server/data/test-suites/live/testfile1.js');
```

Comment:
It'd better to create meanful names for this files, add more semantic, because obviously they describe the different test cases.